src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -73,19 +68,24 @@ const OrganizeTabbedLayout: FunctionComponent<OrganizeTabbedLayoutProps> = ({ ch
                         </Box>
                         <Box p={ 3 } pt={ 1 }>
                             <Typography component="h1" noWrap variant="h2">
-                                { campQuery.data?.title || <Msg id="layout.organize.campaigns.allCampaigns"/> }
+                                { title }
+                            </Typography>
+                            <Typography component="h2" noWrap variant="h5">
+                                { subtitle }
                             </Typography>
                         </Box>
                         <Tabs
                             aria-label="campaign tabs"
-                            onChange={ (_, value) => selectTab(value) }
+                            onChange={ (_, selected) => selectTab(selected) }
                             value={ currentTab }>
-                            { tabs.map(tab => (
-                                <Tab key={ tab } label={ intl.formatMessage({
-                                    id: `layout.organize.campaigns.${tab}`,
-                                }) } value={ tab }
-                                />
-                            )) }
+                            { tabs.map(tab => {
+                                return (
+                                    <Tab key={ tab.label } label={ intl.formatMessage({
+                                        id: `layout.organize.campaigns.${tab.label}`,
```

Comment:
The prefix here is not ideal IMO. It requires that all tabs are below the `layout.organize.campaign` group of messages. Instead, choose one of two options:

1. Let `tab.label` be the human-readable text, and use `intl.formatMessage()` outside of this component to get the localized strings and pass them in.
2. Rename `tab.label` to `tab.messageId` or something like that, and use `intl.formatMessage({ id: tab.messageId }) here.